src/client/features/view/components/tooltips/formatArray.js

```diff
@@ -0,0 +1,20 @@
+const _ = require('lodash');
+
+//Push a metadata field to the bottom of the list
+function pushToBottom(data, field) {
+  let withoutField = _.filter(data, pair => pair[0] !== field);
+  let withField = _.filter(data, pair => pair[0] === field);
+  return _.concat(withoutField, withField);
+}
+
+//Order a given metadata data array
+function orderArray(data) {
```

Comment:
This function still doesn't make sense

Reply:
The purpose was to push a collection of elements to the bottom of a list. 
The name wasn't really that clear and the function wasn't really generic, so I have renamed it and make it slightly more generic.